docs/api_reference/src/quantities.tex

```diff
@@ -146,121 +464,1387 @@
+concept @\defexposconceptnc{same-value-as}@ =
+  equivalent(get_unit(R1), get_unit(R2)) && std::@\stdconcept{convertible_to}@<Rep1, Rep2>;
+
+template<@\libconcept{Reference}@ auto R, @\libconcept{RepresentationOf}@<get_quantity_spec(R).character> Rep = double>
+class @\libglobal{quantity}@ {
+public:
+  Rep @\exposidnc{numerical-value}@;
+
+  // member types and values
+  static constexpr @\libconcept{Reference}@ auto reference = R;
+  static constexpr @\libconcept{QuantitySpec}@ auto quantity_spec = get_quantity_spec(reference);
+  static constexpr @\libconcept{Dimension}@ auto dimension = quantity_spec.dimension;
+  static constexpr @\libconcept{Unit}@ auto unit = get_unit(reference);
+  using rep = Rep;
+
+  // static member functions
+
+  static constexpr quantity zero() noexcept
+    requires requires { quantity_values<rep>::zero(); }
+  {
+    return {quantity_values<rep>::zero(), R};
+  }
+
+  static constexpr quantity one() noexcept
+    requires requires { quantity_values<rep>::one(); }
+  {
+    return {quantity_values<rep>::one(), R};
+  }
+
+  static constexpr quantity min() noexcept
+    requires requires { quantity_values<rep>::min(); }
+  {
+    return {quantity_values<rep>::min(), R};
+  }
+
+  static constexpr quantity max() noexcept
+    requires requires { quantity_values<rep>::max(); }
+  {
+    return {quantity_values<rep>::max(), R};
+  }
+
+  // construction, assignment, destruction
+
+  quantity() = default;
+  quantity(const quantity&) = default;
+  quantity(quantity&&) = default;
+  ~quantity() = default;
+
+  template<typename FwdValue, @\libconcept{Reference}@ R2>
+    requires @\exposconceptnc{same-value-as}@<R2{}, R, [:type_remove_cvref(^FwdValue):], Rep>
+  constexpr quantity(FwdValue&& v, R2) : @\exposidnc{numerical-value}@(std::forward<FwdValue>(v))
+  {
+  }
+};
+
+}
+\end{codeblock}
+
+\pnum
+Let \tcode{\placeholder{Q}} be a specialization of \tcode{quantity}.
+\begin{itemize}
+\item
+If \tcode{Rep} is a scalar\iref{qty.set.traits},
+\tcode{\placeholder{Q}} represents a scalar quantity\irefiev{102-02-19}.
+% FIXME What if `Rep` is a complex number?
+\item
+If \tcode{Rep} is a vector\iref{qty.set.traits},
+\tcode{\placeholder{Q}} represents a vector\irefiev{102-03-04}.
+% FIXME What if `Rep` is a tensor?
+\end{itemize}
+
+\rSec2[qty.pt.concepts]{Quantity point concepts}
+
+\begin{itemdecl}
+template<typename T>
+concept @\deflibconcept{point_origin}@ = @\unspecnc@;
+\end{itemdecl}
+
+\rSec2[qty.pt]{Class template \tcode{quantity_point}}
+
+\begin{codeblock}
+namespace mp_units {
+
+template<@\unspec@>
+class @\libglobal{quantity_point}@ { @\unspec@ };
+
+}
+\end{codeblock}
+
+\pnum
+\label{term.quantity.point.type}
+A \defnadj{quantity point}{type} is a specialization of \tcode{quantity_point}.
+Let \tcode{\placeholder{Q}} be a quantity point type.
+\tcode{\placeholdernc{Q}::point_origin} represents
+the origin point of a position vector\irefiev{102-03-15}
```

Comment:
"position vector" is only one of many quantities specified in ISQ. Please do not use this name here.  It is even hard to talk about position vectors for masses or voltage.

Reply:
It sounds like you think this refers to the quantity.
This refers to yet another definition for the term.
The context is seen in "Area" here: <https://www.electropedia.org/iev/iev.nsf/display?openform&ievref=102-03-15>.

Reply:
Yes, I thought so. But it seems a bit ambiguous. Maybe we can find some alternative spelling?

Reply:
They only have that term for that definition.

There may be a more inclusive definition for the point origin.

Perhaps we could use the definition of [point space](https://www.electropedia.org/iev/iev.nsf/display?openform&ievref=102-03-02).
Something like `Q::point_origin` represents the point whose vector is the [neutral element for addition](https://www.electropedia.org/iev/iev.nsf/display?openform&ievref=102-01-12).

Reply:
...which doesn't work for relative point origins.